src/logic/calculate.test.js

```diff
@@ -0,0 +1,8 @@
+import calculate from './calculate';
+
+describe('calculate', () => {
+  it('should return an object', () => {
+    const result = calculate({ total: 0, next: null, operation: null }, 'AC');
+    expect(result).toEqual({ total: 0, next: null, operation: null });
+  });
+});
```

Comment:
- [ ] Kindly add more tests for the Calculate.js file. There is a lot that you could test for in this file eg of a test you can perform included below:

```
test('test " - " button', () => {
  const obj = {
    total: '100',
    next: '4',
    operation: '-',
  };
  const output = calculate(obj, '=');
  expect(output).toEqual({ total: '96', next: null, operation: null });
});
```